### PR TITLE
Rix1/add content

### PR DESCRIPTION
**Pull request by rix1**
This PR adds content (from the [existing page](http://feriekolonien.no/) to the following pages:

```
src/pages
├── index.js
├── info.js
├── kontakt.js
├── om-oss.js
└── soknadsskjema.js
```

I haven't put much effort into proof-reading nor have I been considering the content itself. This mostly serves as a starting point for further development.

I've also done some restructuring of the components, hopefully making them easier to compose+reuse :v:

## How does it look? 🎥 

Kapture 2019-11-24 at 23 25 11

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      <a className="f6 dib ph2 link mid-gray dim">{children}</a>
+    </Link>
+  );
+};
+
+const Footer = ({ children }) => {
+  return (
+    <footer className="pv4 ph3 ph5-m ph6-l mid-gray">
+      <small className="f6 db tc">
+        <span className="ttu">FILTVET FERIEKOLONI</span>{" "}
```
Comment (rix1) [inline src/components/Footer.js]: We don't need to capitalize this when using `ttu` (`text-transform: uppercase`)

File: src/components/PageComponents.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+// @flow
+export const CTA = ({ children }) => {
+  return (
+    <a
+      className="f4 no-underline grow dib v-mid white ba ph3 pv2 mb3"
+      href="/soknadsskjema"
```
Comment (rix1) [inline src/components/PageComponents.js]: Should probably remove this `href`, wrap it in a `next/link` component and spread props with `{...rest}` to make it easier to reuse.

File: src/components/PageComponents.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+export const HeroContent = ({ children }) => {
+  return <div className="tc-l mt4 mt5-m mt6-l ph3 mw8 center">{children}</div>;
+};
+
+export const Text = ({ children }) => {
+  return <p className="fw3 lh-copy">{children}</p>;
+};
+
+export const PageContent = ({ children, article = false }) => {
```
Comment (rix1) [inline src/components/PageComponents.js]: I don't like the the `article` prop name. It's ambiguous. Do  you have any suggestions @sirihol? What it does is to pull the content "up" and create the "sheet of paper" effect.

image

File: src/components/PageComponents.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    </section>
+  );
+};
+
+export const HeroImage = ({ children }) => {
+  return (
+    <div
+      className="cover bg-left bg-center-l"
+      style={{
+        backgroundImage: "url(/IMG_6409.jpg)"
```
Comment (rix1) [inline src/components/PageComponents.js]: Add `imageUrl` as a prop.

File: src/pages/info.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+import Navigation from "../components/Navigation";
+
+function InfoPage() {
+  return (
+    <Page title="Praktisk info">
+      <HeroImage>
+        <Navigation />
+        <HeroContent>
+          <PageTitle>Dette trenger du å vite</PageTitle>
+          <PageLead></PageLead>
```
Comment (rix1) [inline src/pages/info.js]: Remove unused element.

File: src/components/PageComponents.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
+// @flow
```
Comment (rix1) [inline src/components/PageComponents.js]: Remove unused Flow annotation.

File: src/components/PageComponents.js (hunk 3 above)
Comment (sirihol) [reply src/components/PageComponents.js]: Hmm, funker for meg 😄
Men endrer den bare på stylingen? `paper` eller `paperStyling` ?

File: src/components/AddressComponents.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  return (
+    <AddressLink href={`tel:${children}`} {...rest}>
+      {formatNumber(children, type)}
+    </AddressLink>
+  );
+};
+
+export const Office = ({ children, ...rest }) => {
+  return (
+    <article className="flex-auto w-50">
```
[the remark this answers is not in the document]
Comment (sirihol) [reply src/components/AddressComponents.js]: 🤷🏻‍♀️

**Pull request merged**